src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+    private static final int BALL_NUM = 100;
+    private final int value = new Random().nextInt(BALL_NUM);
```

Comment:
each time you call method, the value will be the same, because it is calculated only once
move calculation to method
```suggestion
    private final Random random = new Random();
```